test/fixtures/roles_users.yml

```diff
@@ -34,3 +34,6 @@ join_00001:
   updated_at: 2009-06-27 18:59:55
   role_id: "6"
   user_id: "1"
+testofficial_official:
+  role_id: 8
+  user_id: 24
```

Comment:
Just a drive-by comment: I think we should also have someone with the opendoors role because (afaik) they have access to tools no other role has.

I don't think the translator role does anything at the moment, so maybe we could drop it from testuser or testuser2 and replace it with opendoors. My initial thought was to give the opendoors role to testuser2 to reduce confusion between the opendoors role and the archivist role, but otoh, it could be convenient to have both roles on one account... 🤔